content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -59,10 +59,18 @@ firewall-cmd --permanent --zone=trusted --add-source=10.43.0.0/16 #services
 firewall-cmd --reload
 ```
 
-You may also need to open other ports if your setup requires them. See the K3s
-[Inbound rules](https://docs.k3s.io/installation/requirements?_highlight=red&_highlight=hat&os=rhel#inbound-rules-for-k3s-nodes)
-docs for more information. If you change the default CIDR for pods or services,
-you must update the firewall rules accordingly.
+You should also add [port rules](https://firewalld.org/documentation/howto/open-a-port-or-service.html)
+for all the [RDI services]({{< relref "/integrate/redis-data-integration/ingest/reference/ports" >}})
+you intend to use:
```

Comment:
that you intend to use

Reply:
@spicy-sauce Adding "that" here isn't a mistake but I've missed it out intentionally here (it's grammatically OK to do so). We try to avoid sounding too formal in the docs.

Reply:
OK, thanks for clarifying.